src/test/java/org/broadinstitute/hellbender/utils/bigquery/BigQueryUtilsUnitTest.java

```diff
@@ -22,12 +21,14 @@
     private static final String BIGQUERY_FULLY_QUALIFIED_TABLE = String.format("%s.%s.%s",
             BIGQUERY_TEST_PROJECT, BIGQUERY_TEST_DATASET, BIGQUERY_TEST_TABLE);
 
+    private static final UUID run_uid = UUID.randomUUID();
```

Comment:
Java Style Guideline -- no underscores in variable names. This should be `runUid`